broccolini.jsx

```diff
@@ -1,330 +0,0 @@
-<meta name='viewport' content='width=device-width,initial-scale=1' />
```

Comment:
Most of this was moved to `pages/index.js` – the viewport meta tag is included in x0 by default